docs/source/Plugin/P017.rst

```diff
@@ -56,6 +56,10 @@ Supported hardware
 
 PN532 based NFC RFID reader connected via I2C on address 0x24.
 
+.. warning::
+
+  This device needs the ``I2C ClockStretchLimit`` (Tools/Advanced page) be set to ``2000`` (msec) to work as intended.

```

Comment:
Maybe this should also be dealt with by the plugin (and every other plugin that needs clock stretch) so the used clock stretch is set to the max. of all enabled tasks.